README.md

```diff
@@ -12,12 +12,12 @@ later.
 
 This demo assumes that [minikube](https://github.com/kubernetes/minikube/) is up and running and `kubectl` available on your system. If not, then please take a look at the guide [below](#appendix-deploying-minikube)
 
-With minikube running, you can easily deploy KubeVirt:
+With minikube *RUNNING*, you can easily deploy KubeVirt:
 
 ```bash
-$ export VERSION=v0.4.1
-$ kubectl create \
-    -f https://github.com/kubevirt/kubevirt/releases/download/$VERSION/kubevirt.yaml
+$ export LATEST_RELEASE_KUBEVIRT_YAML=`curl -s https://api.github.com/repos/kubevirt/kubevirt/releases/latest \
+    | grep browser_download_url | cut -d '"' -f 4 | grep kubevirt.yaml`
+$ kubectl create -f $LATEST_RELEASE_KUBEVIRT_YAML
```

Comment:
I'd actually like to keep the manual updates for now.

Two reasons:
- For the sake of readability I'd like to keep the written out url/version
- When we do version updates, then we also want to update the CI to make sure it is also updated.

Reply:
WRT to your first point - That's okay. I actually wonder if we can keep a latest.txt file around somewhere like Kubernetes does, this could be used here.

Second point - This demo repository has a ci which is testing the demo flow on kube 1.9 and 1.10 every night. This config also needs to be updated for new versions.